keyboards/mechlovin/infinity87/rev2/readme.md

```diff
@@ -14,10 +13,9 @@ Make example for this keyboard (after setting up your build environment):
 
 Flashing example for this keyboard:
 
-    make mechlovin/infinity87/rev2:flash
+    make mechlovin/infinity87/rev2:flashs
```

Comment:
```suggestion
    make mechlovin/infinity87/rev2:flash
```

Reply:
Hi @fauxpark 

The 32A version only made it to a working prototype unit that's currently in our possession, and has never been offered to the public. 

We decided to use 32A on several new designs amidst the global chip shortage, but as new projects come in, we reconsidered our plan and decided to use 32u4 across our TKL lineup. Our apologies for the confusion. We have been taking notes to improve our practice so inconsistencies like this can be avoided in the future. 

Thank you!